cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/DefaultStoreTest.java

```diff
@@ -89,28 +90,22 @@
 /**
  * Tests for {@link DefaultStore}.
  */
-public class DefaultStoreTest {
+public abstract class DefaultStoreTest {
+  protected static DefaultStore store;
+  protected static DefaultNamespaceStore nsStore;
```

Comment:
why is a namespace store and a namespace admin needed? Are they the same thing?

Reply:
We cannot delete the default namespace in namespace admin.